### ISSUE TITLE
nitropy fido2 update fails with NoSoloFoundError('no Nitrokey FIDO2 found') <FIXED: usbguard>

### ISSUE THREAD

**Issue by ek-alliander** [Nitrokey/pynitrokey#65]
I tried to update the firmware of a Nitrokey FIDO2 using nitropy.
The update starts nicely, it detects the USB device, tries to enter bootloader mode, and asks for confirmation with the button on the key.
I press the button on the FIDO2 key, and directly after that the software returns a critical error, and says that no FIDO2 key could be found.
After that the Nitrokey shows a blinking (alternating?) red and blue light. 
I will attach the log that was produced in /tmp.

[nitropy.log.l9gp9su2.txt](https://github.com/Nitrokey/pynitrokey/files/6237846/nitropy.log.l9gp9su2.txt)

**Comment by ek-alliander** [comment]
I am so sorry.
I found the problem, and fixed it.
I am running usbguard, and that blocked the USB device.
When performing the update, and switching to bootloader mode, the USB device ID changes.
The new USB device ID was blocked by usbguard.
I added the second USB device ID to usbguard, and the update was 100% OK.
-> Maybe this is something for the FAQ?

**Comment by daringer** [comment]
closing in favor of: https://github.com/Nitrokey/nitrokey-documentation/issues/190
